jetty-osgi/test-jetty-osgi/src/test/java/org/eclipse/jetty/osgi/test/TestJettyOSGiBootWithAnnotations.java

```diff
@@ -118,16 +119,16 @@ public void testIndex() throws Exception
             assertEquals(HttpStatus.OK_200, response.getStatus());
 
             String content = response.getContentAsString();
-            assertTrue(content.contains("<h1>Servlet 3.1 Test WebApp</h1>"));
+            assertThat(content, containsString("<h1>Servlet 3.0/3.1/4.0 Test WebApp</h1>"));
             
             Request req = client.POST("http://127.0.0.1:" + port + "/test");
             response = req.send();
             content = response.getContentAsString();
-            assertTrue(content.contains("<p><b>Result: <span class=\"pass\">PASS</span></p>"));
+            assertThat(content, containsString("<p><b>Result: <span class=\"pass\">PASS</span></p>"));
             
             response = client.GET("http://127.0.0.1:" + port + "/frag.html");
             content = response.getContentAsString();
-            assertTrue(content.contains("<h1>FRAGMENT</h1>"));
+            assertThat(content, containsString("<h1>FRAGMENT</h1>"));
         }
```

Comment:
Some incidental changes here too, nowt to do with servlet api?

Reply:
reverted.